### DISCUSSION TITLE
What is the current status of Telegraf ?

### DISCUSSION THREAD

**Discussion by raskyer** [telegraf/telegraf#1857] [Q&A]
Hello, 

My team is wondering what is the current status of telegraf ? We consider migrating to another solution in the following weeks because it seems to be unmaintained now ? New api's aren't available and we need it for our business.

Thanks

**Comment by MKRhere** [comment]
We're working on a new release, coming soon (<1 week)! I'd been away for a few months for reasons.

**Reply by raskyer** [reply]
Thanks for your quick answer and hard work ! I hope everything is alright for you 🙏

**Comment by dr-iroh** [comment]
You can temporarily use `bot.telegram.callApi` while the new methods are added, which are only 4 at the moment

```js
bot.telegram.callApi('setMyDescription', options)
bot.telegram.callApi('setMyShortDescription', options)
bot.telegram.callApi('getMyDescription')
bot.telegram.callApi('getMyShortDescription')
```

**Reply by raskyer** [reply]
Not really, with new telegram api you can also set 'webApp' answer to inline response. Which allow to open webapp from any Telegram conversation and I think there is no workaround for now because of typescript definition.

**Reply by raskyer** [reply]
![image](https://github.com/telegraf/telegraf/assets/16725289/ec286f93-947d-41ec-9a5a-ff326e4cb76c)
